src/examples/FloatingUI/useInteractions/UseInteractionsExample.js

```diff
@@ -1,63 +1,71 @@
-import React, {useState} from 'react';
+import React, { useState } from "react";
 
-import {useInteractions, useFloating, useHover, useFocus, useClick, useRole, useDismiss} from '@floating-ui/react';
+import {
+  useInteractions,
+  useFloating,
+  useHover,
+  useFocus,
+  useClick,
+  useRole,
+  useDismiss,
+} from "@floating-ui/react";
 
-const UserInteractionsExample = () => {
-    const [isOpen, setIsOpen] = useState(false);
-    const {context, refs, floatingStyles} = useFloating({
-        open: isOpen,
-        placement: 'top',
-        onOpenChange: setIsOpen,
-    });
+function UserInteractionsExample() {
```

Comment:
its better we follow es6 syntax, this lint config might not work for us

Reply:
That's our next to do item. There are lot of lint errors and warnings that we need to fix. Probably we need a ticket to address all these lint errors and warning